.drone.yml

```
@@ -0,0 +1,105 @@
+---
+kind: pipeline
+type: exec
+name: build-go-release-tarball
+
+concurrency:
+  limit: 1
+
+platform:
+  os: darwin
+  arch: amd64
+
+trigger:
+  event:
+    - tag
+  ref:
+    include:
+      - refs/tags/v*
```

This can be changed as the intention is for the tags here not to start with `v`

```suggestion
  ref:
    include:
      - refs/tags/*
```